h/static/scripts/app.js

```diff
@@ -82,12 +82,17 @@ function configureRoutes($routeProvider) {
   });
 }
 
+function isSidebar() {
+  var path = document.location.pathname;
+  return !path.startsWith('/stream') && !path.startsWith('/a/');
+}
+
 // @ngInject
 function setupFrameSync(frameSync) {
-  // Setup the connection to the frame hosting the sidebar app.
-  // This should only be done if this is the sidebar app, not the stream or
-  // standalone annotation pages.
-  return frameSync.connect();
+  // Setup connection to host frame
+  if (isSidebar()) {
```

Comment:
We already have `isSidebar` defined somewhere else. Shouldn't we move this code somewhere that can take advantage of that rather than redefining a different `isSidebar`?